src/components/SQLEditor/SQLEditor.tsx

```diff
@@ -52,9 +141,11 @@ function SQLEditor({ execCallback, value, results }: Params) {
     if (!sql) {
       return;
     }
-    localStorage.setItem(SQL_EDITOR_CONTENT_KEY, sql);
-    pushHistory(sql);
-    execCallback(sql);
+    if (isLocalStorageUsed) {
```

Comment:
Something broke here with the prev/next navigation - can't seem to use it any more.